modules/special-rules/pages/hitch-x.adoc

```diff
@@ -2,18 +2,18 @@
 
 `Unit ability`
 
-{blank}[TODO: Add introductory description.]
+These units are convenient for friendly but somewhat crazed infantry to grab onto and 'hitch a ride', for transportation across the battlefield.
 
 ---
 
 Where your unit has the {hitch-x} ability:
 
-* The {hitch-x} unit can carry as many Infantry units as we state in brackets.
+* It can transport Infantry units as cargo.
+It can carry as many of these units as we state in brackets.
++
 See the main transportation rules for more about how this works.
 
-NOTE: A {hitch-x} unit does not gain the abilities of embarked units.
-
-.Hitch (X)
+.{hitch-x}
 ====
 The Ork Battlefortress has {hitch-4} -- so you can embark up to four Infantry units in a Battlefortress.
```

Comment:
```suggestion
The Ork Battlefortress has {hitch-4} -- so you can embark up to four Infantry units on a Battlefortress.
```
Just a very minor suggestion, as the point of Hitch is that they never actually get _in_ the vehicle!

Reply:
I agree, I'll apply it and resolve.

There's room to quibble over in/on when it comes to classic open-topped battlewagons and the like, but let's not worry about that! :-D